Packs/MicrosoftGraphSecurity/ReleaseNotes/2_1_19.md

```diff
@@ -0,0 +1,30 @@
+#### Incident Fields
+
+- New: **Microsoft Graph Security Id**
+- New: **Microsoft Graph Security Recommended Action**
+- New: **Microsoft Graph Security Service Source**
+- New: **Microsoft Graph Security Alert Determination**
+- New: **Microsoft Graph Security Comment**
+- New: **Microsoft Graph Security Detector Id**
+- New: **Microsoft Graph Security Alert Evidence**
+
+#### Incident Types
+
+- New: **Graph Security Alert**
+
+#### Integrations
+##### Microsoft Graph Security
+
+- Updated the Docker image to: *demisto/crypto:1.0.0.55191*.
+- Added the command **msg-create-alert-comment**, this command is supported only when using API V2.
+- Added the **MS graph security version** integration param, this param will decide the version of the api to use. Note: the default version is V2 which causes breaking changes and demands reconfiguring with new credentials. If you're using self-deployed application please refer to the readme to the integration readme to the `Required Permissions -> API V2` section to learn about the required permissions. Otherwise, please refer to the link in the `Help` section in the integration configuration window.
```

Comment:
```suggestion
- Added the *MS graph security version* integration parameter. This parameter will determine the version of the API to use. Note: The default version is V2 which causes breaking changes and demands reconfiguring with new credentials. If you're using a self-deployed application, refer to the `Required Permissions -> API V2` section  in the integration readme to learn about the required permissions. Otherwise, refer to the link in the `Help` section in the integration configuration window.
```